src/Iteration.cpp

```diff
@@ -359,6 +430,30 @@ Iteration::read()
     readAttributes();
 }
 
+bool
+Iteration::verifyClosed() const
+{
+    if( dirty )
+    {
+        return false;
```

Comment:
Can you please add three little test cases, e.g. for a single backend, that trigger the false logic from an iteration, particle and mesh?